### DISCUSSION TITLE
Nested apply doesn't seem to work

### DISCUSSION THREAD

**Discussion by songololo** [tailwindlabs/tailwindcss#7366] [Help]
I can't seem to get nested applies to work.

For example, in my `tailwind.css` file I have something along these lines:

```css
@tailwind base;
@tailwind components;
@tailwind utilities;

@layer utilities {
  .my-test-class {
    @apply text-4xl;
  }
}
```

And then if I want to use it from a Vue component, e.g.

```
<style scoped lang="postcss">
.some-class {
  @apply my-test-class;
}
</style>
```
I get this form of error: 
```bash
The `my-test-class` class does not exist. If `my-test-class` is a custom class, make sure it is defined within a `@layer` directive.
```

Any workarounds or ideas? Thanks.

**Comment by gynekolog** [comment]
I'm not sure about Vue but maybe tailwind doc contains answers to your question -> https://tailwindcss.com/docs/adding-custom-styles#layers-and-per-component-css
